_posts/2020-07-18-hackathon.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "LLNL's Summer Hackathon Will Be Virtual"
+categories: event
+---
+
+Held since 2012, LLNL's [hackathons](https://computing.llnl.gov/newsroom/hackathons) are 24-hour opportunities to brainstorm, foster creativity, prototype, and explore. Participants work in groups or individually and often strive to learn new skills, programming languages, and tools in service to LLNL’s missions. Like the spring hackathon earlier this year, the summer event (August 6-7) will be held virtually using WebEx and Mattermost for collaboration. LLNL sponsors are Livermore Computing and the Center for Applied Scientific Computing.
```

Comment:
Do you want to include a link to the sign up page? e.g. hackathon.llnl.gov ?

Reply:
I didn't know that was a real URL. I'll add it with the lock icon.